### PR TITLE
Improve RQT visualisers (STF) text descriptions 

### PR DESCRIPTION
**Pull request by rnzenha-s**
## Proposed changes

Improve RQT sensor visualisers (STF) text descriptions.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py [hunk 1]
```diff
@@ -273,17 +273,20 @@ def __init__(self, side, parent):
...
         self.launch_plotjuggler_button = QPushButton("Launch PlotJuggler*")
         self.launch_plotjuggler_button.clicked.connect(self._button_action_launch_plotjuggler)
         layout.addWidget(self.launch_plotjuggler_button, alignment=Qt.AlignCenter)
 
-        note_label = QLabel("*Note: You'll need to click 'Yes' to Start the Streaming and select"
-                            f" /{self._side}/tactile data topic.")
-        layout.addWidget(note_label, alignment=Qt.AlignCenter)
+        note_label_1 = QLabel(" *You'll need to click 'Yes' to Start the Streaming and select"
+                              f" /{self._side}/tactile data topic.")
+        note_label_2 = QLabel("Please note that the Plotjuggler tools has been known to crash."
```
Comment (toliver) [inline sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py]: singular "tool"

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py (hunk 1 above)
Comment (rnzenha-s) [reply sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py]: Fixed!

**Review by toliver — APPROVED** [review]

File: sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py [hunk 2]
```diff
@@ -273,17 +273,20 @@ def __init__(self, side, parent):
         layout = QVBoxLayout()
 
         layout.setAlignment(Qt.AlignVCenter)
-        no_tactile_label = QLabel("\t\tSTF sensors not supported yet."
-                                  " \nAlternatively, you can launch the PlotJuggler data inspection tool:")
+        no_tactile_label = QLabel("\t      STF sensors are not supported yet."
```
Comment (dg-shadow) [inline sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py]: mix of tabs and spaces seems odd

Comment (rnzenha-s) [reply sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py]: Fixed now! https://github.com/shadow-robot/sr-visualization/pull/317/commits/f00ed1e6670662820df5d7b1aabfbaff73d30ca8

**Review by dg-shadow — APPROVED** [review]

**Pull request merged**